File: WebApi/InWords.WebApi/InWords.Transfer.Data/Models/GameBox/GameInfo.cs
namespace InWords.Transfer.Data.Models.GameBox
{
    public class GameInfo
    {
        public int GameId { get; set; }

        public int CreatorId { get; set; }

        public string Description { get; set; }

        public string Title { get; set; }

        public bool IsAvailable { get; set; }
    }
}

File: Android/app/src/main/java/ru/inwords/inwords/data/dto/game/GameInfo.kt
package ru.inwords.inwords.data.dto.game

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(tableName = "game_info_table")
data class GameInfo(
        @PrimaryKey
        @SerializedName("gameId") val gameId: Int,
        @SerializedName("creatorId") val creatorId: Int,
        @SerializedName("description") val description: String,
        @SerializedName("title") val title: String,
        @SerializedName("isAvaliable") val available: Boolean)